### ISSUE TITLE
Errors getting blocks on Kusama

### ISSUE THREAD

**Issue by joepetrowski** [paritytech/substrate-api-sidecar#253]
Node: v0.8.23
Sidecar: cb0114c3ce8d723e76b10df50cff453d0f4ba85a

Lots of issues with some recent blocks:
```
GET /block/3860498 200 89.430 ms - 2542
GET /block/3860499 200 105.013 ms - 1403
GET /block/3860500 200 140.271 ms - 1267
GET /block/3860501 200 123.557 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 640123136 exceeds 32768
2020-09-01 20:33:34        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 640123136 exceeds 32768
GET /block/3860502 200 94.861 ms - 2023
GET /block/3860503 200 83.428 ms - 2563
GET /block/3860504 200 90.262 ms - 1403
GET /block/3860505 200 84.059 ms - 1075

GET /block/3860965 200 120.164 ms - 1075
GET /block/3860966 200 107.344 ms - 1403
GET /block/3860967 200 119.523 ms - 1075
GET /block/3860968 200 102.427 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 1020887296 exceeds 32768
2020-09-01 20:34:21        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 1020887296 exceeds 32768
GET /block/3860969 200 82.530 ms - 1889
GET /block/3860970 200 81.501 ms - 1075
GET /block/3860971 200 84.792 ms - 1595
GET /block/3860972 200 108.438 ms - 1267
GET /block/3860973 200 112.822 ms - 1403

GET /block/3861060 200 108.970 ms - 1075
GET /block/3861061 200 108.838 ms - 1595
GET /block/3861062 200 96.271 ms - 1595
GET /block/3861063 200 37.912 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 842006784 exceeds 32768
2020-09-01 20:34:30        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 842006784 exceeds 32768
GET /block/3861064 200 75.661 ms - 2981
GET /block/3861065 200 39.170 ms - 1403
GET /block/3861066 200 40.160 ms - 1595
GET /block/3861067 200 82.534 ms - 1403
GET /block/3861068 200 99.353 ms - 2239
GET /block/3861069 200 94.699 ms - 1403

GET /block/3861261 200 109.493 ms - 1075
GET /block/3861262 200 109.284 ms - 1595
GET /block/3861263 200 45.398 ms - 1403
GET /block/3861264 200 84.991 ms - 2207
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 505872640 exceeds 32768
2020-09-01 20:34:49        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 505872640 exceeds 32768
GET /block/3861265 200 147.669 ms - 3015
GET /block/3861266 200 85.006 ms - 1267
GET /block/3861267 200 100.381 ms - 1403
GET /block/3861268 200 94.864 ms - 1403
GET /block/3861269 200 55.283 ms - 1075

GET /block/3861289 200 91.210 ms - 1595
GET /block/3861290 200 40.403 ms - 1403
GET /block/3861291 200 78.940 ms - 1075
GET /block/3861292 200 118.153 ms - 1595
Unable to decode storage system.events: createType(Vec<EventRecord>):: Unable to create Enum via index 84, in ApplyExtrinsic, Finalization, Initialization
2020-09-01 20:34:52        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Unable to create Enum via index 84, in ApplyExtrinsic, Finalization, Initialization
GET /block/3861293 200 133.070 ms - 8712
GET /block/3861294 200 37.126 ms - 1075
GET /block/3861295 200 81.738 ms - 1075
GET /block/3861296 200 110.011 ms - 1403
GET /block/3861297 200 96.032 ms - 1075
GET /block/3861298 200 89.161 ms - 1595
GET /block/3861299 200 110.838 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Unable to create Enum via index 187, in ApplyExtrinsic, Finalization, Initialization
2020-09-01 20:34:52        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Unable to create Enum via index 187, in ApplyExtrinsic, Finalization, Initialization
GET /block/3861300 200 132.304 ms - 7807
GET /block/3861301 200 81.364 ms - 1403
GET /block/3861302 200 97.466 ms - 1403
GET /block/3861303 200 89.907 ms - 1403

GET /block/3861348 200 96.787 ms - 1403
GET /block/3861349 200 111.991 ms - 2365
GET /block/3861350 200 965.842 ms - 308621
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 428048640 exceeds 32768
2020-09-01 20:34:59        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 428048640 exceeds 32768
GET /block/3861351 200 119.311 ms - 14766
GET /block/3861352 200 79.625 ms - 3209
GET /block/3861353 200 98.871 ms - 1267
GET /block/3861354 200 106.875 ms - 1403
GET /block/3861355 200 79.776 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 939196672 exceeds 32768
2020-09-01 20:34:59        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 939196672 exceeds 32768
GET /block/3861356 200 78.960 ms - 3447
GET /block/3861357 200 84.186 ms - 1075
GET /block/3861358 200 96.474 ms - 1403

GET /block/3861388 200 84.185 ms - 1403
GET /block/3861389 200 101.774 ms - 2153
GET /block/3861390 200 99.296 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 20705280115911769920947946632091389780138979597267728640922859925403166 exceeds 32768
2020-09-01 20:35:02        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 20705280115911769920947946632091389780138979597267728640922859925403166 exceeds 32768
GET /block/3861391 200 64.463 ms - 2761
GET /block/3861392 200 81.529 ms - 1075
GET /block/3861393 200 113.553 ms - 1403
GET /block/3861394 200 87.368 ms - 1403

GET /block/3861538 200 81.427 ms - 1075
GET /block/3861539 200 79.117 ms - 1403
GET /block/3861540 200 88.767 ms - 1075
GET /block/3861541 200 100.119 ms - 1595
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 805601536 exceeds 32768
2020-09-01 20:35:17        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 805601536 exceeds 32768
GET /block/3861542 200 84.002 ms - 2200
GET /block/3861543 200 80.212 ms - 1075
GET /block/3861544 200 87.583 ms - 1403
GET /block/3861545 200 119.666 ms - 1595
GET /block/3861546 200 83.661 ms - 1075

GET /block/3861646 200 79.232 ms - 2269
GET /block/3861647 200 92.115 ms - 1267
GET /block/3861648 200 94.701 ms - 1595
GET /block/3861649 200 113.616 ms - 1595
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 255951104 exceeds 32768
2020-09-01 20:35:27        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 255951104 exceeds 32768
GET /block/3861650 200 94.205 ms - 2654
GET /block/3861651 200 85.372 ms - 1403
GET /block/3861652 200 92.571 ms - 2039
GET /block/3861653 200 84.850 ms - 1595
GET /block/3861654 200 80.656 ms - 1403
GET /block/3861655 200 81.876 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 887390464 exceeds 32768
2020-09-01 20:35:28        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 887390464 exceeds 32768
GET /block/3861656 200 81.622 ms - 2392
GET /block/3861657 200 81.432 ms - 1075
GET /block/3861658 200 117.573 ms - 1403
GET /block/3861659 200 112.190 ms - 1403

GET /block/3861704 200 106.209 ms - 1075
GET /block/3861705 200 81.864 ms - 1403
GET /block/3861706 200 46.498 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 646840576 exceeds 32768
2020-09-01 20:35:33        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 646840576 exceeds 32768
GET /block/3861707 200 80.005 ms - 1878
GET /block/3861708 200 88.293 ms - 1595
GET /block/3861709 200 54.133 ms - 1403
GET /block/3861710 200 81.259 ms - 1075
GET /block/3861711 200 116.227 ms - 1595
GET /block/3861712 200 88.066 ms - 1403
GET /block/3861713 200 78.478 ms - 1075
GET /block/3861714 200 41.877 ms - 1403
GET /block/3861715 200 76.943 ms - 1075
GET /block/3861716 200 78.088 ms - 1595
GET /block/3861717 200 86.475 ms - 1403
GET /block/3861718 200 84.518 ms - 1075
GET /block/3861719 200 91.922 ms - 1595
GET /block/3861720 200 97.438 ms - 1403
GET /block/3861721 200 85.346 ms - 1075
GET /block/3861722 200 118.939 ms - 1403
GET /block/3861723 200 67.655 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 578388224 exceeds 32768
2020-09-01 20:35:34        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 578388224 exceeds 32768
GET /block/3861724 200 62.196 ms - 2669
GET /block/3861725 200 74.928 ms - 1267
GET /block/3861726 200 101.122 ms - 1595

GET /block/3861770 200 80.057 ms - 1595
GET /block/3861771 200 84.267 ms - 1403
GET /block/3861772 200 78.808 ms - 1928
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 610894080 exceeds 32768
2020-09-01 20:35:38        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 610894080 exceeds 32768
GET /block/3861773 200 80.605 ms - 2333
GET /block/3861774 200 103.997 ms - 1267
GET /block/3861775 200 164.256 ms - 1595
GET /block/3861776 200 129.489 ms - 1403
GET /block/3861777 200 95.903 ms - 1892
GET /block/3861778 200 38.420 ms - 1595
GET /block/3861779 200 82.945 ms - 1403
GET /block/3861780 200 124.713 ms - 1075
GET /block/3861781 200 93.771 ms - 1595
GET /block/3861782 200 46.698 ms - 1403
GET /block/3861783 200 84.613 ms - 1075
GET /block/3861784 200 112.431 ms - 1403
GET /block/3861785 200 102.396 ms - 1075
GET /block/3861786 200 87.435 ms - 1403
GET /block/3861787 200 93.452 ms - 1267
GET /block/3861788 200 74.553 ms - 2448
GET /block/3861789 200 81.568 ms - 1403
GET /block/3861790 200 85.674 ms - 1075
GET /block/3861791 200 75.816 ms - 1403
GET /block/3861792 200 94.463 ms - 1403
GET /block/3861793 200 100.419 ms - 1075
GET /block/3861794 200 123.935 ms - 1595
GET /block/3861795 200 110.011 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 640123136 exceeds 32768
2020-09-01 20:35:41        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 640123136 exceeds 32768
GET /block/3861796 200 104.056 ms - 3257
GET /block/3861797 200 79.734 ms - 1403
GET /block/3861798 200 91.806 ms - 1403
GET /block/3861799 200 53.504 ms - 1403
GET /block/3861800 200 96.796 ms - 1403
GET /block/3861801 200 111.445 ms - 1075

GET /block/3861821 200 117.162 ms - 1595
GET /block/3861822 200 99.765 ms - 1267
GET /block/3861823 200 104.397 ms - 1403
GET /block/3861824 200 83.049 ms - 1403
GET /block/3861825 200 87.902 ms - 2125
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 23757056 exceeds 32768
2020-09-01 20:35:44        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 23757056 exceeds 32768
GET /block/3861826 200 85.220 ms - 4107
GET /block/3861827 200 101.299 ms - 1267
GET /block/3861828 200 65.642 ms - 1403
GET /block/3861829 200 45.793 ms - 1595
GET /block/3861830 200 120.425 ms - 1075
GET /block/3861831 200 101.807 ms - 2348
GET /block/3861832 200 88.450 ms - 1267
GET /block/3861833 200 38.895 ms - 1595
GET /block/3861834 200 82.768 ms - 1403
GET /block/3861835 200 90.969 ms - 1075
GET /block/3861836 200 108.773 ms - 2220
GET /block/3861837 200 84.962 ms - 1595
GET /block/3861838 200 93.104 ms - 1928
GET /block/3861839 200 60.352 ms - 2564
GET /block/3861840 200 89.035 ms - 1595
GET /block/3861841 200 107.046 ms - 1403
GET /block/3861842 200 103.229 ms - 1595
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 732070144 exceeds 32768
2020-09-01 20:35:45        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 732070144 exceeds 32768
GET /block/3861843 200 118.230 ms - 2704
GET /block/3861844 200 82.405 ms - 1403
GET /block/3861845 200 101.533 ms - 1403
GET /block/3861846 200 132.733 ms - 1075
GET /block/3861847 200 129.617 ms - 1403
GET /block/3861848 200 142.348 ms - 1075
GET /block/3861849 200 84.760 ms - 2267
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 164987136 exceeds 32768
2020-09-01 20:35:46        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 164987136 exceeds 32768
GET /block/3861850 200 88.133 ms - 2823
GET /block/3861851 200 109.973 ms - 1075
GET /block/3861852 200 79.988 ms - 1403
GET /block/3861853 200 89.773 ms - 1075

GET /block/3861932 200 79.301 ms - 1075
GET /block/3861933 200 113.547 ms - 1595
GET /block/3861934 200 83.533 ms - 1403
GET /block/3861935 200 78.124 ms - 1075
GET /block/3861936 200 83.816 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 181764352 exceeds 32768
2020-09-01 20:35:54        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 181764352 exceeds 32768
GET /block/3861937 200 125.069 ms - 2010
GET /block/3861938 200 82.527 ms - 1595
GET /block/3861939 200 111.561 ms - 1403
GET /block/3861940 200 88.598 ms - 1075

GET /block/3861981 200 36.329 ms - 1075
GET /block/3861982 200 77.505 ms - 1595
GET /block/3861983 200 44.683 ms - 1595
GET /block/3861984 200 114.713 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 971899136 exceeds 32768
2020-09-01 20:35:59        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 971899136 exceeds 32768
GET /block/3861985 200 92.740 ms - 2512
GET /block/3861986 200 113.302 ms - 1075
GET /block/3861987 200 107.261 ms - 1403

GET /block/3862074 200 95.193 ms - 1595
GET /block/3862075 200 78.615 ms - 1075
GET /block/3862076 200 95.674 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 907739392 exceeds 32768
2020-09-01 20:36:08        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 907739392 exceeds 32768
GET /block/3862077 200 78.845 ms - 2346
GET /block/3862078 200 103.716 ms - 1595
GET /block/3862079 200 96.110 ms - 1595

GET /block/3862118 200 82.034 ms - 1075
GET /block/3862119 200 84.006 ms - 2564
GET /block/3862120 200 81.623 ms - 1403
Unable to decode storage system.events: createType(Vec<EventRecord>):: Unable to create Enum via index 23, in ApplyExtrinsic, Finalization, Initialization
2020-09-01 20:36:12        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Unable to create Enum via index 23, in ApplyExtrinsic, Finalization, Initialization
GET /block/3862121 200 152.167 ms - 7192
GET /block/3862122 200 70.600 ms - 2440
GET /block/3862123 200 78.459 ms - 1075
GET /block/3862124 200 72.848 ms - 1403
GET /block/3862125 200 31.060 ms - 1075
GET /block/3862126 200 33.163 ms - 1403
GET /block/3862127 200 32.378 ms - 1595
GET /block/3862128 200 29.071 ms - 1075
GET /block/3862129 200 33.568 ms - 1403
GET /block/3862130 200 32.923 ms - 1595
GET /block/3862131 200 29.705 ms - 1267
GET /block/3862132 200 33.194 ms - 1403
GET /block/3862133 200 72.162 ms - 1267
GET /block/3862134 200 72.354 ms - 1403
GET /block/3862135 200 71.250 ms - 1403
GET /block/3862136 200 70.133 ms - 1075
GET /block/3862137 200 67.858 ms - 1403
GET /block/3862138 200 72.830 ms - 1075
GET /block/3862139 200 74.092 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Unable to create Enum via index 217, in ApplyExtrinsic, Finalization, Initialization
2020-09-01 20:36:13        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Unable to create Enum via index 217, in ApplyExtrinsic, Finalization, Initialization
GET /block/3862140 200 88.081 ms - 8129
GET /block/3862141 200 70.394 ms - 1075
GET /block/3862142 200 70.573 ms - 1403
GET /block/3862143 200 78.981 ms - 1403
GET /block/3862144 200 78.155 ms - 1403
GET /block/3862145 200 71.858 ms - 1595
GET /block/3862146 200 78.256 ms - 1075
GET /block/3862147 200 86.656 ms - 2375
Unable to decode storage system.events: createType(Vec<EventRecord>):: Unable to create Enum via index 157, in ApplyExtrinsic, Finalization, Initialization
2020-09-01 20:36:14        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Unable to create Enum via index 157, in ApplyExtrinsic, Finalization, Initialization
GET /block/3862148 200 81.832 ms - 8129
GET /block/3862149 200 71.881 ms - 1403
GET /block/3862150 200 43.944 ms - 1403
GET /block/3862151 200 75.898 ms - 1595
GET /block/3862152 200 73.090 ms - 1075
GET /block/3862153 200 72.125 ms - 1403
GET /block/3862154 200 72.721 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 994509056 exceeds 32768
2020-09-01 20:36:14        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 994509056 exceeds 32768
GET /block/3862155 200 76.841 ms - 7150
GET /block/3862156 200 74.895 ms - 1267
GET /block/3862157 200 84.016 ms - 1403

GET /block/3862187 200 131.417 ms - 1595
GET /block/3862188 200 107.306 ms - 1403
GET /block/3862189 200 86.889 ms - 1075
GET /block/3862190 200 79.595 ms - 1403
GET /block/3862191 200 88.774 ms - 1075
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 876380416 exceeds 32768
2020-09-01 20:36:18        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 876380416 exceeds 32768
GET /block/3862192 200 87.354 ms - 2979
GET /block/3862193 200 121.786 ms - 1403
GET /block/3862194 200 101.929 ms - 1403
GET /block/3862195 200 108.329 ms - 1403

GET /block/3862232 200 104.717 ms - 1403
GET /block/3862233 200 115.830 ms - 1075
GET /block/3862234 200 90.416 ms - 49807
Unable to decode storage system.events: createType(Vec<EventRecord>):: Vec length 1002242304 exceeds 32768
2020-09-01 20:36:22        RPC-CORE: getStorage(key: StorageKey, at?: BlockHash): StorageData:: createType(Vec<EventRecord>):: Vec length 1002242304 exceeds 32768
GET /block/3862235 200 85.645 ms - 3796
GET /block/3862236 200 75.528 ms - 1075
GET /block/3862237 200 93.742 ms - 1403
GET /block/3862238 200 94.454 ms - 1403
```

**Comment by joepetrowski** [comment]
Calling these blocks individually on master and v0.16.0 works fine. I'm increasingly thinking that there's a client issue when running for a long time on Kusama.

**Comment by emostov** [comment]
At the moment my main issue is just trying to get this error reproduced. I will look into creating a branch  that logs the actual bytes received.

**Comment by joepetrowski** [comment]
OK, not looking at bytes, but did get a block that failed and then worked after restart.

Failure:
```
{'number': '3888505',
 'hash': '0xd180e2f3d96d80ca676bf7cef8a9f57f61bae483589d77400181082ae4419d48',
 'parentHash': '0xbcd5a687e749351e59e6de6d8753789559b6aae6f4c0a5c4ae620d3123988580',
 'stateRoot': '0x735c4b6cd3ee24a1da0306a3be605f26289ac10e60d9c05e8f3a635a572c1215',
 'extrinsicsRoot': '0x675ffd053e447d302d269495e0c58cf15f332905ad1de94558fb588fd4bbfd5c',
 'authorId': 'CdvyGjJbfcgcHuTyJKLGUA9u3kypCaHpohCEeUXmbTKryPR',
 'logs': [{'type': 'PreRuntime',
   'index': '6',
   'value': ['BABE',
    '0x017800000091d5e20f00000000f444f6f34655deee9c6bdaa332a4b713b7b3576bd3f21225c075abd41d9a5d6245664befe371987031e3da5ca351d27ec3769f7ee12ee812ffa389af8a0ca001b64ae9693171bb8b3e24fb3a9886c9c961e82d8244a5440bb802bfc9d59e8f0f']},
  {'type': 'Seal',
   'index': '5',
   'value': ['BABE',
    '0x5c692937183f28722e594afc8cd82cd6671849f8cdeda9cd62a256111e4e0001a60d7a09a026388772cc7b64b2cdfba042a028a6adbc4ce236648c9ae93c7188']}],
 'onInitialize': {'events': []},
 'extrinsics': [{'method': 'timestamp.set',
   'signature': None,
   'nonce': '0',
   'args': {'now': '1599144294000'},
   'tip': '0',
   'hash': '0x2b808df63e4be1382bf9f6b0499e66e03c6db2bf75da3b9209126f93acde6339',
   'info': {},
   'events': [],
   'success': 'Unable to fetch Events, cannot confirm extrinsic status. Check pruning settings on the node.',
   'paysFee': None},
  {'method': 'finalityTracker.finalHint',
   'signature': None,
   'nonce': '0',
   'args': {'hint': '3888502'},
   'tip': '0',
   'hash': '0x13d6a917aec8d867b0938daab72dd3d0c5f6ec4b8abc3606b2d08e2e0a59a0f9',
   'info': {},
   'events': [],
   'success': 'Unable to fetch Events, cannot confirm extrinsic status. Check pruning settings on the node.',
   'paysFee': None},
  {'method': 'utility.batch',
   'signature': {'signature': '0x6cd73f3be5f3d40d33c93d53a43f2f46f8a408f293433abd2f5f3f75036c996f893e6e482fc7977885e14746049822927c1f5cf6fa7b29dfcb3aee366ef1d683',
    'signer': 'CgjHH1YNfNeBzQawgqcfFVFYTL67CbVZ1p8pwxDbKmWxR8S'},
   'nonce': '4',
   'args': {'calls': [{'method': 'staking.payoutStakers',
      'callIndex': {'0': '6', '1': '18'},
      'args': {'validator_stash': 'CzugcaJpsC977jZ6cjfTpQxBkptNLNxUieW2Kj7qZoXpco1',
       'era': '1235'}},
     {'method': 'staking.payoutStakers',
      'callIndex': {'0': '6', '1': '18'},
      'args': {'validator_stash': 'CzugcauySs7eZ5im4y7dgLAZzGZkXLfNCirRNGjxbNvwJyh',
       'era': '1236'}}]},
   'tip': '0',
   'hash': '0xe96b6dbcc944b8d502927b2a0934ac64853807aa90d38d175ccc7c1ab8267f95',
   'info': {},
   'events': [],
   'success': 'Unable to fetch Events, cannot confirm extrinsic status. Check pruning settings on the node.',
   'paysFee': None}],
 'onFinalize': {'events': []}}
```

Success:
```
{'number': '3888505',
 'hash': '0xd180e2f3d96d80ca676bf7cef8a9f57f61bae483589d77400181082ae4419d48',
 'parentHash': '0xbcd5a687e749351e59e6de6d8753789559b6aae6f4c0a5c4ae620d3123988580',
 'stateRoot': '0x735c4b6cd3ee24a1da0306a3be605f26289ac10e60d9c05e8f3a635a572c1215',
 'extrinsicsRoot': '0x675ffd053e447d302d269495e0c58cf15f332905ad1de94558fb588fd4bbfd5c',
 'authorId': 'CdvyGjJbfcgcHuTyJKLGUA9u3kypCaHpohCEeUXmbTKryPR',
 'logs': [{'type': 'PreRuntime',
   'index': '6',
   'value': ['BABE',
    '0x017800000091d5e20f00000000f444f6f34655deee9c6bdaa332a4b713b7b3576bd3f21225c075abd41d9a5d6245664befe371987031e3da5ca351d27ec3769f7ee12ee812ffa389af8a0ca001b64ae9693171bb8b3e24fb3a9886c9c961e82d8244a5440bb802bfc9d59e8f0f']},
  {'type': 'Seal',
   'index': '5',
   'value': ['BABE',
    '0x5c692937183f28722e594afc8cd82cd6671849f8cdeda9cd62a256111e4e0001a60d7a09a026388772cc7b64b2cdfba042a028a6adbc4ce236648c9ae93c7188']}],
 'onInitialize': {'events': []},
 'extrinsics': [{'method': 'timestamp.set',
   'signature': None,
   'nonce': '0',
   'args': {'now': '1599144294000'},
   'tip': '0',
   'hash': '0x2b808df63e4be1382bf9f6b0499e66e03c6db2bf75da3b9209126f93acde6339',
   'info': {},
   'events': [{'method': 'system.ExtrinsicSuccess',
     'data': [{'weight': '159133000',
       'class': 'Mandatory',
       'paysFee': 'Yes'}]}],
   'success': True,
   'paysFee': True},
  {'method': 'finalityTracker.finalHint',
   'signature': None,
   'nonce': '0',
   'args': {'hint': '3888502'},
   'tip': '0',
   'hash': '0x13d6a917aec8d867b0938daab72dd3d0c5f6ec4b8abc3606b2d08e2e0a59a0f9',
   'info': {},
   'events': [{'method': 'system.ExtrinsicSuccess',
     'data': [{'weight': '0', 'class': 'Mandatory', 'paysFee': 'Yes'}]}],
   'success': True,
   'paysFee': True},
  {'method': 'utility.batch',
   'signature': {'signature': '0x6cd73f3be5f3d40d33c93d53a43f2f46f8a408f293433abd2f5f3f75036c996f893e6e482fc7977885e14746049822927c1f5cf6fa7b29dfcb3aee366ef1d683',
    'signer': 'CgjHH1YNfNeBzQawgqcfFVFYTL67CbVZ1p8pwxDbKmWxR8S'},
   'nonce': '4',
   'args': {'calls': [{'method': 'staking.payoutStakers',
      'callIndex': {'0': '6', '1': '18'},
      'args': {'validator_stash': 'CzugcaJpsC977jZ6cjfTpQxBkptNLNxUieW2Kj7qZoXpco1',
       'era': '1235'}},
     {'method': 'staking.payoutStakers',
      'callIndex': {'0': '6', '1': '18'},
      'args': {'validator_stash': 'CzugcauySs7eZ5im4y7dgLAZzGZkXLfNCirRNGjxbNvwJyh',
       'era': '1236'}}]},
   'tip': '0',
   'hash': '0xe96b6dbcc944b8d502927b2a0934ac64853807aa90d38d175ccc7c1ab8267f95',
   'info': {'weight': '62772425000',
    'class': 'Normal',
    'partialFee': '3216665529'},
   'events': [{'method': 'staking.Reward',
     'data': ['CzugcaJpsC977jZ6cjfTpQxBkptNLNxUieW2Kj7qZoXpco1',
      '28727508096']},
    {'method': 'staking.Reward',
     'data': ['CgjHH1YNfNeBzQawgqcfFVFYTL67CbVZ1p8pwxDbKmWxR8S', '481466040']},
    {'method': 'staking.Reward',
     'data': ['D2y4de3DrNwWwf51voWHuUv8o7uxLLYMmMuS2NNmhJrYugf',
      '5543345328']},
    {'method': 'staking.Reward',
     'data': ['FUZhtSaXWKqEo9nySW3D6hQAuck6YRiau9tjswRpDYc3urM',
      '922371472252']},
    {'method': 'staking.Reward',
     'data': ['CzugcauySs7eZ5im4y7dgLAZzGZkXLfNCirRNGjxbNvwJyh',
      '24091733231']},
    {'method': 'staking.Reward',
     'data': ['GtREhyeZyB59TPXFjggKXrnDta82FKFbfp1QSYnKNf9Lyac',
      '23694516702']},
    {'method': 'staking.Reward',
     'data': ['Fy2yG79zSHGRWfgCbHMVdH4Vq4iY2qRkmhLpVDmt1vbvE4X',
      '4134225813']},
    {'method': 'staking.Reward',
     'data': ['CgjHH1YNfNeBzQawgqcfFVFYTL67CbVZ1p8pwxDbKmWxR8S', '399188673']},
    {'method': 'staking.Reward',
     'data': ['HFcwc88LyQt3bfA2MpexBsBTGZAKvRGN7rt65W6aPYQwYf5',
      '1270005063']},
    {'method': 'staking.Reward',
     'data': ['DaQ9vYVF9ZCrYSVydYikYeWF1qAPTw1JQewyyZNQzATBBjr',
      '59824291206']},
    {'method': 'staking.Reward',
     'data': ['FLQv1YKitnBJ8BpXassBgA88jVjekdffT9PPr7RVa9u2CWe',
      '333677354117']},
    {'method': 'staking.Reward',
     'data': ['G27mJ9MkcjqnLpsNi3oHzcpzUKaBzGSLBpVMQa8K6RhLs2d',
      '116642724089']},
    {'method': 'staking.Reward',
     'data': ['FZfup4GH5fNDPvMqmooo84xs24XYadevJY1trBDrk8yKHjg',
      '2291078086']},
    {'method': 'staking.Reward',
     'data': ['D39ZKxumrn4QUsQZBVAghponcXasUjSAppNCECkuCTq5tPy', '64553421']},
    {'method': 'staking.Reward',
     'data': ['Da1rxnb6Envzn9bpCkU1DFRFCuT4KFXQML4pDmBGvFrbjnH',
      '91476008644']},
    {'method': 'staking.Reward',
     'data': ['H7Piypdvnyw8ewwnSvHvQvVJJmH1jM4MSaYt1sjSVpR9XbZ',
      '145110941004']},
    {'method': 'utility.BatchCompleted', 'data': []},
    {'method': 'treasury.Deposit', 'data': ['2573332423']},
    {'method': 'balances.Deposit',
     'data': ['CdvyGjJbfcgcHuTyJKLGUA9u3kypCaHpohCEeUXmbTKryPR', '643333106']},
    {'method': 'system.ExtrinsicSuccess',
     'data': [{'weight': '62772425000',
       'class': 'Normal',
       'paysFee': 'Yes'}]}],
   'success': True,
   'paysFee': True}],
 'onFinalize': {'events': []}}
```

**Comment by emostov** [comment]
This should be fixed with #291
